packages/@stimulus/core/src/context.ts

```diff
@@ -70,6 +70,13 @@ export class Context implements ErrorHandler {
     return this.element.parentElement
   }
 
+  dispatch(eventName: String, { target = this.element, detail = {}, bubbles = true, cancelable = true } = {}) {
+    const type = `${this.identifier}:${eventName}`
```

Comment:
I really like this! Maybe we could also make the `prefix` customizable? What do you think?

```suggestion
  dispatch(eventName: String, { target = this.element, detail = {}, prefix = this.identifier, bubbles = true, cancelable = true } = {}) {
    const type = prefix ? `${prefix}:${eventName}` : eventName
```

Reply:
Is `prefix` the best name to use here? I've been using `namespace`, but maybe `identifier` is more appropriate?

Reply:
@dancallaghan I also like `namespace` or `identifier`. 

The reason I suggested  `prefix` is because `stimulus-use` has been using that for a while: https://github.com/stimulus-use/stimulus-use/blob/master/docs/use-dispatch.md#mixin

But I would say that all of the three options work well.